h2o-algos/src/main/java/hex/generic/Generic.java

```diff
@@ -81,17 +82,20 @@ public void computeImpl() {
                 final MojoReaderBackend readerBackend = MojoReaderBackendFactory.createReaderBackend(mojoBytes.openStream(_job._key), MojoReaderBackendFactory.CachingStrategy.MEMORY);
                 mojoModel = ModelMojoReader.readFrom(readerBackend, true);
                 
-                if(!ALLOWED_MOJO_ALGOS.contains(mojoModel._modelDescriptor.algoName().toLowerCase())){
-                    throw new IllegalArgumentException(String.format("Unsupported MOJO model '%s'. ", mojoModel._modelDescriptor.algoName()));
+                if(! ALLOWED_MOJO_ALGOS.contains(mojoModel._modelDescriptor.algoName().toLowerCase())) {
```

Comment:
Scoring is going to work for all MOJOs. I have a different strategy on my mind - just allowing all the algos and only warning the user that model metrics are not available. Not necessarily to be solved in this PR, just a thought to work with :)

Reply:
As you're using "Dummy metrics builder", we could simply make scoring work for every MOJO and only display the warning, as you wrote it.

Reply:
Interesting, I think that is a good idea.